automates/text_reading/src/main/scala/org/clulab/aske/automates/mentions/CrossSentenceEventMention.scala

```diff
@@ -0,0 +1,41 @@
+package org.clulab.aske.automates.mentions
+
+import org.clulab.odin
+import org.clulab.odin.{Mention, _}
+import org.clulab.processors.Document
+import org.clulab.struct.Interval
+
+// note: same as EventMention but has args from two different sentences - thus requires two sentence information.
+// fixme: this is not properly deserialized yet.
+
+class CrossSentenceEventMention(
+                                 val labels: Seq[String],
+                                 val tokenInterval: Interval,
+                                 val trigger: TextBoundMention,
+                                 val arguments: Map[String, Seq[Mention]],
+                                 val paths: Map[String, Map[Mention, SynPath]],
+                                 val sentence: Int,
+                                 val sentences: Seq[Int],
+                                 val document: Document,
+                                 val keep: Boolean,
+                                 val foundBy: String,
+                                 val attachments: Set[Attachment]
+                               ) extends Mention {
+
+  def this(
+            label: String,
+            trigger: TextBoundMention,
+            arguments: Map[String, Seq[Mention]],
+            paths: Map[String, Map[Mention, SynPath]],
+            sentence: Int,
+            sentences: Seq[Int],
+            document: Document,
+            keep: Boolean,
+            foundBy: String
+          ) = this(Seq(label), mkTokenInterval(trigger, arguments), trigger, arguments, paths, sentence, sentences, document, keep, foundBy, Set.empty)
+}
+
+object CrossSentenceEventMention
```

Comment:
Did you want to make a new text method for this type of event that will make a text span from arguments? Or you c could just borrow the one from Eidos (where this class came from---let me know if you need help refinding it). The generic "text" method will probably not get a good cross-sentence text. I can help with this (should be fast for me, up to you)

Reply:
It would be great if it's possible to get a text span from arguments, but they are in two different sentences... Would it be possible to get a text span over two sentences?

Reply:
The args should have the information about the correct sentence. I don't think it should be a problem.

Reply:
You're right. Can we discuss this at tomorrow's meeting?